MeshLib/Elements/EdgeReturn.cpp

```diff
@@ -0,0 +1,49 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2015, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "EdgeReturn.h"
+
+#include "logog/include/logog.hpp"
+
+#include "MeshLib/Node.h"
+#include "Element.h"
+#include "Line.h"
+
+namespace MeshLib
+{
+
+const Element* LinearEdgeReturn::getEdge(const Element* e, unsigned i)
```

Comment:
Note: The linear and quadratic implementations return a Line or Line3, which are derived from Edge, so the return type could be `Edge*` too (consistently with the function name getEdge).